spec/stepmod/smrl_description_converter_spec.rb

```diff
@@ -33,6 +33,6 @@
         <express_ref linkend="classification_and_set_theory:ir:classification_schema.class" />
       XML
     )
-    expect(described_class.convert(input)).to eq("<<express:classification_schema:classification_schema.class,class>>")
+    expect(described_class.convert(input)).to eq("<<express:classification_schema.class,class>>")
```

Comment:
@w00lf I believe we want just `<<express:classification_schema.class>>`.

@opoudjis is this correct?

Reply:
No. We do want that text to be "class", because the text otherwise will be the full path, "classification_schema.class"

Reply:
OK